Site/PullRequestPreviewBranchWorkflow.md

```diff
@@ -0,0 +1,132 @@
+# A Proposed Pull Request Workflow using a Preview Branch
+
+The primary goal of this workflow is to allow the usage of a single BSSw
+GitHub branch and Pull Request (PR) to be created and edited by multiple
+actors where all reviews, edits, and all other modifications are made in that
+one branch/PR before the PR is merged to the `master` branch and deployed on
+the `bssw.io` site and to allow for previews on `preview.bssw.io` without
+needing to merge to `master` first.  This is in contrast to current workflow
+where the draft content is merged to `master` very early and subsequent edits
+are done un-reviewed on `master`.
+
+
+## Actors in the workflow
+
+* **Author:** Creates a branch with the first draft of the proposed content on
+  a branch (i.e. `content-A` in their fork of the BSSw repo) and then creates
+  a PR of that branch against BSSw `master`.  (This could be actual content
+  author or the editorial board member that collects the initial draft of the
+  contribution from from Google Form placed there by the original author.)
```

Comment:
What is the author experience if they do not have write access to the repo?  They needs to fork, develop their content locally and create a PR for our repo.  But then how does the iteration on the PR work for them? (I have no experience with this.)

Reply:
> What is the author experience if they do not have write access to the repo? 

If the actual author does not use the fork and PR model, then they can't edit the PR branch.  All they can do is ask the EB member to do it.  This is what we saw in PR #424 and it is very unfortunate.  If the original author is comfortable with github, we need to strongly recommend they fork, create branch, and create PR so they can push updates to the branch.  Other options  are possible but I don't even want to describe them hear to avoid making people's head spin.

Reply:
I understand that this is really the only way for the author to interact.  I was trying to get more at the details of what the iterative refinement process looks like from the author's perspective at the git command level.